arangod/Transaction/Manager.cpp

```diff
@@ -105,11 +104,8 @@ void Manager::registerTransaction(TransactionId transactionId,
   if (!isReadOnlyTransaction && !isFollowerTransaction) {
     LOG_TOPIC("ccdea", TRACE, Logger::TRANSACTIONS)
         << "Acquiring read lock for tid " << transactionId.id();
-    _rwLock.lockRead();
-    _nrReadLocked.fetch_add(1, std::memory_order_relaxed);
     LOG_TOPIC("ccdeb", TRACE, Logger::TRANSACTIONS)
-        << "Got read lock for tid " << transactionId.id()
-        << " nrReadLocked: " << _nrReadLocked.load(std::memory_order_relaxed);
+        << "Got read lock for tid " << transactionId.id();
```

Comment:
This code is identical to the code in devel (which should be the source of truth), but effectively the `if` condition now only guards two log messages. There is no lock acquired here anymore, so the log messages are IMO misleading and can be removed.